src/hotspot/share/runtime/globals_ext.hpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 1997, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Alibaba Group Holding Limited. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+#ifndef SHARE_RUNTIME_GLOBALS_EXT_HPP
+#define SHARE_RUNTIME_GLOBALS_EXT_HPP
+
+#define DRAGONWELL_FLAGS(develop,                                              \
+                         develop_pd,                                           \
+                         product,                                              \
+                         product_pd,                                           \
+                         notproduct,                                           \
+                         range,                                                \
+                         constraint)                                           \
+  product(bool, UseNewCode4, false, DIAGNOSTIC,                                \
```

Comment:
what's this flag used for?

Reply:
It's used for testing.

Reply:
Does adding a UseNewCode4 here offer anything different from the 3 existing UseNewCode globals?

Reply:
No usage of this option. It's used to test dragonwell specific flag.